site/content/docs/manifest/environment.en.md

```diff
@@ -0,0 +1,177 @@
+Must be set to `'Environment'`.
+
+<div class="separator"></div>
+
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>  
+The network section contains parameters for importing an existing VPC or configuring the Copilot generated VPC.
+
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>  
+The vpc section contains parameters to configure CIDR settings and subnets.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-id" href="#network-vpc-id" class="field">`id`</a> <span class="type">String</span>    
+The ID of the VPC to import. This field is mutually exclusive with `cidr`.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-cidr" href="#network-vpc-cidr" class="field">`cidr`</a> <span class="type">String</span>    
+An IPv4 CIDR block to associate with the Copilot-generated VPC. This field is mutually exclusive with `id`.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-subnets" href="#network-vpc-subnets" class="field">`subnets`</a> <span class="type">Map</span>    
+Configure public and private subnets in a VPC.
+
+For example, if you're importing an existing VPC:
+```yaml
+network:
+  vpc:
+    public:
+      - id: 'subnet-11111'
+      - id: 'subnet-22222'
+```
+Alternatively, if you're configuring a Copilot-generated VPC:
+```yaml
+network:
+  vpc:
+    cidr: '10.0.0.0/16'
+    subnets:
+      public:
+        - cidr: '10.0.0.0/24'
+          az: 'us-east-2a'
+        - cidr: '10.0.1.0/24'
+          az: 'us-east-2b'
+```
+
+<span class="parent-field">network.vpc.subnets.</span><a id="network-vpc-subnets-public" href="#network-vpc-subnets-public" class="field">`public`</a> <span class="type">Array of Subnets</span>    
+A list of public subnets configuration.
+
+<span class="parent-field">network.vpc.subnets.</span><a id="network-vpc-subnets-private" href="#network-vpc-subnets-private" class="field">`private`</a> <span class="type">Array of Subnets</span>    
+A list of private subnets configuration.
+
+<span class="parent-field">network.vpc.subnets.<type\>.</span><a id="network-vpc-subnets-id" href="#network-vpc-subnets-id" class="field">`id`</a> <span class="type">String</span>    
+The ID of the subnet to import. This field is mutually exclusive with `cidr` and `az`.
+
+<span class="parent-field">network.vpc.subnets.<type\>.</span><a id="network-vpc-subnets-cidr" href="#network-vpc-subnets-cidr" class="field">`cidr`</a> <span class="type">String</span>    
+An IPv4 CIDR block assigned to the subnet. This field is mutually exclusive with `id`.
+
+<span class="parent-field">network.vpc.subnets.<type\>.</span><a id="network-vpc-subnets-az" href="#network-vpc-subnets-az" class="field">`az`</a> <span class="type">String</span>    
+The AZ name assigned to the subnet. This field is mutually exclusive with `id`.
```

Comment:
can you set _only_ `cidr` or `az`? or do they always go together

Reply:
Fixed! 

> The Availability Zone name assigned to the subnet. The `az` field is optional, by default Availability Zones are assigned in alphabetical order.
This field is mutually exclusive with `id`.